src/domain/collaboration/canvas/containers/canvasQueries.graphql

```diff
@@ -108,6 +134,22 @@ query hubCanvasValues($hubId: UUID_NAMEID!, $calloutId: UUID_NAMEID!, $canvasId:
   }
 }
 
+query challengeCanvasFromCallout(
+  $hubId: UUID_NAMEID!
+  $challengeId: UUID_NAMEID!
+  $calloutId: UUID_NAMEID!
+  $canvasId: UUID_NAMEID!
+) {
+  hub(ID: $hubId) {
+    id
+    challenge(ID: $challengeId) {
+      id
+      collaboration {
+        ...CalloutWithCanvas
+      }
+    }
+  }
+}
 query challengeCanvases($hubId: UUID_NAMEID!, $challengeId: UUID_NAMEID!) {
```

Comment:
Can you remove these if we are not using them anymore? I think we are still using hubCanvasesLazyQuery in the admin section and I'm not sure how hard would it be to use these new ones, but I think for Challenges and Opportunities we don't use them anymore

Reply:
Yes, removed challengeCanvases and opportunityCanvases queries.